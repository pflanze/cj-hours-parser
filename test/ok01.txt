# this is a comment
[
this is a multiline
  comment
]
-1<!--this is some subitem comment-->2:44 tes<!--and another one -- > -->t  some time


13:27-18:44 test-again -->

