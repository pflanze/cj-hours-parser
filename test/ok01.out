item=SingleLineComment(Owned(" this is a comment"))
    =SingleLineComment(Owned(" this is a comment"))
item=MultiLineComment([Owned("this is a multiline"), Inline("  comment")])
    =MultiLineComment([Owned("this is a multiline"), Inline("  comment")])
item=Other([Minus, Time(NT(12:44:00)), Rest(Inline("test  some time"))])
    =Other([Subitem(L(0, 6, 0), Minus), Subitem(L(0, 6, 1), Time(NT(12:44:00))), Subitem(L(0, 6, 42), Rest(Inline("test  some time")))])
item=Empty(2)
    =Empty(2)
item=Other([Time(NT(13:27:00)), Minus, Time(NT(18:44:00)), Rest(Inline("test-again -->"))])
    =Other([Subitem(L(0, 9, 0), Time(NT(13:27:00))), Subitem(L(0, 9, 5), Minus), Subitem(L(0, 9, 6), Time(NT(18:44:00))), Subitem(L(0, 9, 12), Rest(Inline("test-again -->")))])
item=Empty(1)
    =Empty(1)
Hello, world!
